.readthedocs.yml

```diff
@@ -0,0 +1,19 @@
+# .readthedocs.yaml
+# Read the Docs configuration file
+# See https://docs.readthedocs.io/en/stable/config-file/v2.html for details
+
+# Required
+version: 2
+
+build:
+  os: ubuntu-20.04
```

Comment:
I guess this is the ubuntu version circleci uses in it's docker containers?  (which is annoyingly different than we use in our docker containers - debian buster vs ubuntu 20.04, based on debian bullseye)

Reply:
Yeah, it matches CI (which annoyingly has its own docker images, but at least it's the same Ubuntu version). I wish our docker images were based on the same version, but at least this is consistent with CI.